src/lib/hooks/provider-hooks.ts

```diff
@@ -9,15 +9,17 @@ export function useEagerConnect() {
 	const [tried, setTried] = useState(false);
 
 	useEffect(() => {
-		injected.isAuthorized().then((isAuthorized: boolean) => {
-			if (isAuthorized) {
-				activate(injected, undefined, true).catch(() => {
+		if (localStorage.getItem('chosenWallet')) {
```

Comment:
What about non-metamask wallets? Could use the local storage key `chosenWallet` to instead be metamask, portis, walletconnect, or other wallets and try to auto reconnect with those? (Instead of just being 'yes')